src/optimizer/limit_pushdown.cpp

```diff
@@ -0,0 +1,42 @@
+#include "duckdb/optimizer/limit_pushdown.hpp"
+
+#include "duckdb/planner/operator/logical_limit.hpp"
+#include "duckdb/planner/operator/logical_projection.hpp"
+#include <iostream>
```

Comment:
nit: don't think we need to include upstream here

Reply:
done!